### PR TITLE
[RHACS] Added instructions for static network policy generator

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-12107

Preview: https://openshift-docs-git-rox-12107-netpol-gnelson.vercel.app/openshift-acs/master/operating/manage-network-policies.html#using-static-network-policy-generator

### CODE REVIEW

File: modules/generate-static-network-policies.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+* Refer to the link:https://github.com/stackrox/stackrox#build-tooling[Build tooling] section of the StackRox repository.
+
+.Procedure
+. Clone the StackRox repository locally and open the `stackrox` directory.
++
+[source,terminal]
+----
+$ git https://github.com/stackrox/stackrox.git && cd stackrox
+----
+. Generate the `roxctl` CLI binary for your operating system:
```
Comment (vikin91) [inline modules/generate-static-network-policies.adoc:26]: "Generate" is okay, but we are actually compiling it in this step. Could we maybe use verbs like: "build", or "compile"?

**Review by vikin91 — APPROVED** [review]
Technically sounds very good, but pls take a look at my comment.

File: modules/generate-static-network-policies.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/generate-static-network-policies.adoc:26]: Thank you @vikin91 I'll update.

**Review by Wiharris — COMMENTED** [review]
/LGTM

**Pull request merged**